<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncidenceDateTimeEditor</class>
 <widget class="QWidget" name="IncidenceDateTimeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>174</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="mStartLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Start:</string>
       </property>
       <property name="buddy">
        <cstring>mStartDateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mStartCheck">
       <property name="whatsThis">
        <string>Sets the start date for this to-do</string>
       </property>
       <property name="text">
        <string extracomment="@option:check to-do start datetime">Sta&amp;rt:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QDateEdit" name="mStartDateEdit">
     <property name="displayFormat">
      <string>MM.dd.yyyy</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QTimeEdit" name="mStartTimeEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="mEndLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;End:</string>
       </property>
       <property name="buddy">
        <cstring>mEndDateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mEndCheck">
       <property name="whatsThis">
        <string>Sets the due date for this to-do.</string>
       </property>
       <property name="text">
        <string extracomment="to-do due datetime">&amp;Due:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QDateEdit" name="mEndDateEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>MM.dd.yyyy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QTimeEdit" name="mEndTimeEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="mFreeBusyLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Show time as:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="mFreeRadio">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mBusyRadio">
       <property name="text">
        <string>Busy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="KPIM::KTimeZoneComboBox" name="mTimeZoneComboStart">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Select the timezone for this event. It will also affect recurrences</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="mHasTimeCheck">
     <property name="whatsThis">
      <string>Set if this to-do's start and due dates have times associated with them.</string>
     </property>
     <property name="text">
      <string>Ti&amp;me associated</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="KPIM::KTimeZoneComboBox" name="mTimeZoneComboEnd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Select the timezone for this event. It will also affect recurrences</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPIM::KTimeZoneComboBox</class>
   <extends>KComboBox</extends>
   <header>libkdepim/ktimezonecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
